### PR TITLE
Add config migrator to the update tab

### PR DESCRIPTION
**Pull request by BelleNottelling**
This can be used by FOSSBilling users when manually updating, or when they are migrating from BoxBilling.
image

### CODE REVIEW

**Comment by BelleNottelling** [comment]
Also just created a new test for the config migrator; it executes the tool, ensures that it completes, and then compares the before & after config arrays to ensure they are the same

File: src/bb-themes/admin_default/html/mod_extension_index.html.twig [hunk 1]
```diff
@@ -109,6 +109,17 @@
                 </a>
                 <hr>
 
+                <h3>{{ 'Migrate configuration file'|trans }}</h3>
+                <p class="text-muted">{{ 'Use this too to automatically migrate your configuration file to be compatible with the latest FOSSBilling changes. Note: this is only needed when performing manual updates'|trans }}</p>
```
Comment (evrifaessa) [inline src/bb-themes/admin_default/html/mod_extension_index.html.twig:113]: too -> tool

**Review by evrifaessa — APPROVED** [review]
Great PR :)

**Comment by John-S4** [comment]
Makes sense and anything that simplifies, and means less talking people though, BB migrations has to be a good thing.

**Review by John-S4 — APPROVED** [review]

**Pull request merged**